src/commands/build/index.js

```diff
@@ -27,7 +25,7 @@ class BuildCommand extends Command {
   }
 
   checkOptions({ cachedConfig: { siteInfo = {} }, token }) {
-    if (!siteInfo.id && process.env.NODE_ENV !== 'test') {
```

Comment:
We are now passing a fake site id in the tests so we can remove this test specific code